src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,33 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
-
+        int[] arr = new int[2];
+        try (BufferedReader buffer = new BufferedReader(new FileReader(fromFileName))) {
+            String line = buffer.readLine();
+            while (line != null) {
+                if (line.split(",")[0].equals("supply")) {
```

Comment:
let's create constants for 0 and 1 array indexes
```suggestion
                if (line.split(",")[OPERATION_INDEX].equals("supply")) {
```